### PR TITLE
A new organization using groups

### PR DESCRIPTION
**Pull request by contraexemplo**
Thought it would be a good idea to create a PR to discuss this in particular (and maybe already merge it to make further work easier):

This PR creates an explicit thematic sorting for our documentation using the groups feature (more on that here: https://docs.gitbook.com/content-editing/pages-structure#using-groups). `Internal` was moved to be the last group of pages because I think it's the less interesting of them for the person who is looking for help immediately.

This doesn't change any  file except `SUMMARY.md`.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -17,12 +23,18 @@
   * [Core Contributors: Expenses](the-open-collective-way/core-contributors-expenses.md)
```
Comment (alanna) [inline SUMMARY.md]: Maybe these 'core contributors' sections should go in Internal?

**Comment by alanna** [comment]
Fiscal hosts need their own section, not under Financial Contributors. It's a seperate thing.

**Comment by alanna** [comment]
"Expenses & Getting Paid" is also a seperate thing, not a type of Financial Contribution.

**Review by jaskiratsingh2000 — COMMENTED** [review]
I agree with @alanna above. Rest everything looks fine. Good to go !

**Comment by contraexemplo** [comment]
@alanna and @jaskirat2000 I addressed the changes you requested:
- `Fiscal Hosts` and `Expenses & Getting Paid` now have their own group of pages
- Pages related to `Core Contributors` were moved to `Internal`

And a final suggestion:
- Moving `Values` from `The Open Collective Way` to `About`. `Community Guidelines` is already a part of `About` so it seems to fit there better.

**Pull request merged**